core/src/main/java/org/apache/druid/math/expr/BinaryLogicalOperatorExpr.java

```diff
@@ -27,7 +27,7 @@
 import java.util.Objects;
 
 // logical operators live here
-
+@SuppressWarnings("ClassName")
```

Comment:
I guess the issue is there's no public class BinaryLogicalOperatorExpr?

I suppose that's strange, but fine.

Reply:
right, and it also expects each class to be in its own file with the same name